tests/integration/test_lists/test-db/l0_dgx_b200_perf_sanity.yml

```diff
@@ -15,9 +15,7 @@ l0_dgx_b200_perf_sanity:
       backend: pytorch
       orchestrator: mpi
   tests:
-  - perf/test_perf.py::test_perf[perf_sanity_aggr_upload-deepseek_r1_fp8_blackwell-r1_fp8_dep8_mtp1_1k1k] TIMEOUT (180)
-  - perf/test_perf.py::test_perf[perf_sanity_aggr_upload-deepseek_r1_fp8_blackwell-r1_fp8_tep8_mtp3_1k1k] TIMEOUT (180)
-  - perf/test_perf.py::test_perf[perf_sanity_aggr_upload-deepseek_r1_fp8_blackwell-r1_fp8_tp8_mtp3_1k1k] TIMEOUT (180)
+  - perf/test_perf_sanity.py::test_e2e[aggr_upload-deepseek_r1_fp8_blackwell-1-3] TIMEOUT (180)
```

Comment:
Why do we change `r1_fp8_dep8_mtp1_1k1k` to `1-3`?
I think using a label might be more stable/reproduce than using the case index? Because cases might be deleted or added, thus the index can change. How do you think?